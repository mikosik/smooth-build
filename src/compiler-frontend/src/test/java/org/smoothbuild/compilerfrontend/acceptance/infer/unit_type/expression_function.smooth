Int myFunc(A a) = 7;
result = myFunc([]);
