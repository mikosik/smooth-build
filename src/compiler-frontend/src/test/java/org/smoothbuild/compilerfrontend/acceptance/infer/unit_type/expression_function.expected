Int myFunc<A>(A a)
  = 7;
Int result<>
  = myFunc<[{}]>([]);
